src/components/CreateInstructorPage/CreateInstructorForm.jsx

```diff
@@ -0,0 +1,169 @@
+        <div className="col">
+          <form onSubmit={handleSubmit}>
+            <Field
+              name="image"
+              component={ImageUpload}
+              label={<React.Fragment>Image: <span className="required">*</span></React.Fragment>}
+              id="image"
+              required
+            />
+            <Field
+              name="firstName"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  First name <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="lastName"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  Last name <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="title"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  Title <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="organization"
+              component={RenderSelectField}
+              options={extractOrgChoices(instructorOptions)}
+              label={
+                <React.Fragment>
+                  Organization
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="bio"
+              component={RichEditor}
+              label={
+                <React.Fragment>
+                  Biography <span className="required">*</span>
+                </React.Fragment>
+              }
+              maxChars={250}
+              validate={basicValidate}
+              id="bio"
+            />
+            <Field
+              name="majorWorks"
+              component={RichEditor}
+              label={
+                <React.Fragment>
+                  Major works (optional)
+                </React.Fragment>
+              }
+              maxChars={250}
+              id="works"
+            />
+            <FieldArray
+              name="socialLinks"
+              component={SocialLinks}
+            />
+            <FieldArray
+              name="areasOfExpertise"
+              component={AreasOfExpertise}
+            />
+            <div className="row justify-content-end">
+              <Link
+                className={['btn btn-link ml-3 form-cancel-btn']}
+                to="/"
+                disabled={formControlDisabled}
+                onClick={() => {
+                  store.dispatch(push('/'));
```

Comment:
hmm, this will need to be cleaned up in the "instructor workflow" story, but good enough for now.